index.html

```diff
@@ -347,5 +347,104 @@ <h2>Examples</h2>
       </pre>
     </div>
   </section>
+  <section>
+    <h2>Transferable MediaStreamTrack</h2>
+    <div>
+      <p>{{MediaStreamTrack}} are <a data-cite="!HTML/#transferable-objects">transferable objects</a>.
+      This allows manipulating real-time media outside the context it was requested or created, for instance in workers or third-party iframes.</p>
+      <p>To preserve the existing privacy and security infrastructure, in particular for capture tracks,
+      a transferred track remains tightly bound to the original track.
+      In particular, if the original track gets muted, unmuted or ended, the transferred track will automatically be muted, unmuted or ended.
```

Comment:
I find this sentence confusing, as no "original track" survives a transfer in any functional sense to have experiences like "gets muted" (only a "detached" interface remains).

I also think the (un)muted and enabled properties are implicit in being a `MediaStreamTrack` (we say less in [clone](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediastream-clone)).

But I think we need to expressly limit the track being [live](https://w3c.github.io/mediacapture-main/getusermedia.html#idl-def-MediaStreamTrackState.live) to when the realm it was originally granted to exists. And this mustn't be transitive (if the transferred track is transferred yet again). How about:
```suggestion
      <p>To preserve existing privacy, security, and permission infrastructure, in particular for tracks from
      capture sources, transferred tracks MUST [= track/ended | end =], if they haven't already, when they
      would have ended had they never been transferred.
```

This also exposes that mediacapture-main doesn't expressly say anywhere that tracks must end on navigation. We should probably specify that with some [unloading document cleanup steps](https://html.spec.whatwg.org/multipage/browsing-the-web.html#unloading-document-cleanup-steps). — Luckily [browsers already do this](https://bugzil.la/1685163).